### PR TITLE
Fix some golint failures of pkg/kubectl/cmd/attach pkg/kubectl/cmd/autoscale

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fix golint failures of pkg/kubectl/cmd/attach pkg/kubectl/cmd/autoscale

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by ahmadnurus — COMMENTED** [review]
renaming `AttachOptions` to `Options` is not a good approach. it has been rejected in the past.

File: pkg/kubectl/cmd/attach/attach.go [hunk 1]
```diff
@@ -61,8 +61,8 @@ const (
 	defaultPodLogsTimeout   = 20 * time.Second
 )
 
-// AttachOptions declare the arguments accepted by the Exec command
-type AttachOptions struct {
+// Options declare the arguments accepted by the Attach command
+type Options struct {
```
Comment (prksu) [inline pkg/kubectl/cmd/attach/attach.go]: don't change these.

Comment (SataQiu) [reply pkg/kubectl/cmd/attach/attach.go]: @prksu Thanks for your feedback.
This is the default recommendation of `golint`.
Is there a better way to solve this problem?
I need more helpful advice.

Comment (prksu) [reply pkg/kubectl/cmd/attach/attach.go]: i once asked this before in the slack sig-cli channel. and it has been rejected before.
my advice just leave the current naming. see https://github.com/kubernetes/kubernetes/pull/70682#discussion_r233642556

Comment (SataQiu) [reply pkg/kubectl/cmd/attach/attach.go]: Thanks for your explanation. That make sense!

Comment (SataQiu) [reply pkg/kubectl/cmd/attach/attach.go]: Code updated. @prksu

**Pull request merged**